test/e2e/utils/ui/surveyForm.js

```diff
@@ -0,0 +1,58 @@
+import { click, getElement, toRightOf, writeIntoTextBox } from '../api'
+import { waitForLoader } from './loader'
+
+const selectors = {
+  name: () => toRightOf('Name'),
+  label: () => toRightOf('Label'),
+  key: () => toRightOf('Key'),
+  multiple: () => toRightOf('Multiple'),
+}
```

Comment:
aren't these related to node def details? 
better to separate nodeDefDetails from surveyForm

Reply:
Perfect!